Makefile

```diff
@@ -8,6 +8,18 @@ ifneq ($(wildcard ./.git/.),)
 VERSION := $(shell git describe --tags)
 endif
 
+SERVICEDIR_DBUS ?= $(shell pkg-config dbus-1 --variable=session_bus_services_dir)
```

Comment:
I'm not sure about this one, dbus dirs are standardized in `dbus-daemon(1)` and using this ensures that any custom prefix (e.g. `~/.local/share`) won't get the dbus service installed.

Reply:
You can set `SERVICEDIR_DBUS` by yourself and therefore you can specify your own folder.